bug_report.php

```diff
@@ -341,6 +344,11 @@
 	array( string_get_bug_view_url( $t_bug_id ), sprintf( lang_get( 'view_submitted_bug_link' ), $t_bug_id ) ),
 	array( 'view_all_bug_page.php', lang_get( 'view_bugs_link' ) ),
 );
+
+if ( $t_share_link != '' ) {
+	$t_buttons[] = array( $t_share_link, lang_get( 'share_this_bug' ) );
```

Comment:
I would drop this button and again limit it to a Share button on the View Issue page that shows a page that discloses the URL.